File: 37/Tennis.py
from Campi import Campi

class Tennis(Campi):
    def __init__(self, codice, tipo, nomeCampo, larghezza, lunghezza, costoOrario,tempMedia,terreno) -> None:
        super().__init__(codice, tipo, nomeCampo, larghezza, lunghezza, costoOrario)
        self.tempMedia = tempMedia
        self.terreno   = terreno

    def __str__(self) -> str:
        return self.tipo+"\t"+self.nomeCampo+"\t"+str(self.codice)+"\t"+str(self.larghezza)+"\t"+str(self.lunghezza)+"\t"+str(self.tempMedia)+"\t"+self.terreno+"\t-"+"\t-\t"+str(self.costoOrario)
    

File: Sim02 -Campi Con Split/prova - SENZAsplit/Tennis.java
/*
 * @ Perrotta Marco
 * 
 * Struttura Tennis figlia di Campo 
 * 
 */

public class Tennis extends Campo {
    float temperatura_media;
    String terreno;

    public Tennis(int codice, String sport, String nomeCampo, float larghezza, float lunghezza, float costo_orario,
            float temperatura_media, String terreno) {
        super(codice, sport, nomeCampo, larghezza, lunghezza, costo_orario);
        this.temperatura_media = temperatura_media;
        this.terreno = terreno;
    }
                        
    public String toString() {
        return sport + " " + nomeCampo + " " + codice + " " + larghezza + " " + lunghezza + " " + temperatura_media
                + " " + terreno + " - " + " - " + costo_orario;
    }

}
